pysmt/simplifier.py

```diff
@@ -31,6 +31,9 @@ def __init__(self, env=None):
         self.functions[op.BOOL_CONSTANT] = self.walk_identity
         self.functions[op.INT_CONSTANT] = self.walk_identity
 
+        for bvop in op.BV_OPERATORS:
+            self.functions[bvop] = self.walk_identity
```

Comment:
Isn't there any obvious simplification (e.g. bvadd(0 x), bvmult(1, x), ...) ? Maybe we can postpone this and open an issue